target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -262,6 +265,9 @@ ath79_setup_macs()
 	local board="$1"
 
 	case "$board" in
+	airtight,c65)
+		lan_mac=$(macaddr_setbit_la "$(mtd_get_mac_binary art 0)")
```

Comment:
Why are you setting the local bit for lan here (&eth0 is set to art 0x0 already)?

Reply:
To me it looks like that line can be dropped.